source/features/remove-label-button.css

```diff
@@ -0,0 +1,21 @@
+.rgh-remove-label-button {
+	display: flex !important;
+	margin-left: 2px;
+	margin-right: -7px;
+	color: inherit !important;
+}
+
+.rgh-remove-label-button svg {
+	margin: 2px;
+	margin-left: 0;
+	border-radius: 50%;
+	padding: 2px 0;
+	width: 14px !important;
+	height: 14px !important;
```

Comment:
I'm not sure that `!important` is required here. I think CSS properties override SVG attributes